_caster.py

```diff
@@ -112,6 +112,8 @@ def generate_sm_ccr_choices(nexus):
             R(Function(navigation.mouse_alternates, mode="rainbow", nexus=_NEXUS)),
         "douglas [<monitor>]":
             R(Function(navigation.mouse_alternates, mode="douglas", nexus=_NEXUS)),
+        "sudoku [<monitor>]":
```

Comment:
This is been re-factored into `castervoice\rules\core\utility_rules\mouse_alts_rules.py` and no longer uses the nexus.

```Python
        "douglas [<monitor>]":
            R(Function(navigation.mouse_alternates, mode="douglas")),
```